### PR TITLE
Map

### PR DESCRIPTION
**Pull request by BaranekD**

### CODE REVIEW

File: api/src/main/java/cz/metacentrum/perun/spRegistration/persistence/configs/AppConfig.java [hunk 1]
```diff
@@ -254,6 +255,17 @@ public void setPerunAttributeDefinitionsMap(Map<String, PerunAttributeDefinition
 		this.perunAttributeDefinitionsMap = perunAttributeDefinitionsMap;
 	}
 
+	public Map<String, String> getMails() { return mails; }
```
Comment (dBucik) [inline api/src/main/java/cz/metacentrum/perun/spRegistration/persistence/configs/AppConfig.java]: Change "mails" to something more descriptive.  We are using this map as a register for the institution -> authority email pairing, when user requests moving service into the production environment.
I would suggest something like prodTransferAuthoritiesMailsMap

File: api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java [hunk 2]
```diff
@@ -10,10 +10,7 @@
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.SessionAttribute;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (dBucik) [inline api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java]: Do not use asterisk wildcard. Rather import each dependency separately. (Probably problem in IDE settings)

File: api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java [hunk 3]
```diff
@@ -100,4 +97,14 @@ public boolean getSpecifyAuthoritiesEnabled() {
 		log.trace("getSpecifyAuthoritiesEnabled() returns: {}", specifyAuthoritiesEnabled);
 		return specifyAuthoritiesEnabled;
 	}
+
+	@GetMapping(path = "/api/config/mailsMap")
```
Comment (dBucik) [inline api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java]: Apply rename here as well for both method and mapping

File: api/src/main/resources/application-context.xml [hunk 4]
```diff
@@ -161,6 +158,14 @@
         <property name="jdbcTemplate" ref="jdbcTemplate"/>
     </bean>
 
+    <bean id="mailsMap" class="org.springframework.beans.factory.config.PropertiesFactoryBean" >
+        <property name="locations">
+            <list>
+                <value>file:///etc/perun-spRegistration/mailsMap.properties</value>
```
Comment (dBucik) [inline api/src/main/resources/application-context.xml]: Also here change the name of the file to something more descriptive.

File: api/src/main/java/cz/metacentrum/perun/spRegistration/persistence/configs/AppConfig.java (hunk 1 above)
Comment (BaranekD) [reply api/src/main/java/cz/metacentrum/perun/spRegistration/persistence/configs/AppConfig.java]: Done

File: api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java (hunk 2 above)
Comment (BaranekD) [reply api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java]: Done

File: api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java (hunk 3 above)
Comment (BaranekD) [reply api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java]: Done

File: api/src/main/resources/application-context.xml (hunk 4 above)
Comment (BaranekD) [reply api/src/main/resources/application-context.xml]: Done

**Pull request merged**